Shared/Extensions/StringExtensions.swift

```diff
@@ -96,4 +96,16 @@ public extension String {
         let cleaned = self.replacingOccurrences(of: "\n", with: " ")
         return cleaned.trimmingCharacters(in: .whitespacesAndNewlines).components(separatedBy: string)
     }
+    
+    public func startsWith(_ other: String) -> Bool {
```

Comment:
Can't we just use: https://developer.apple.com/documentation/swift/string/2894028-starts

Reply:
We don't even use it, removed.